config/locales/devise.en.yml

```diff
@@ -0,0 +1,62 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+en:
```

Comment:
@keyurgohil2426 We should always put all labels in yml files, better for localization. no need to do it now.